Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/serialization/SerializationModule.kt

```diff
@@ -59,8 +73,6 @@ class SerializationModule {
                 // Others
                 .registerTypeAdapter(ByteArray::class.java, ByteArrayAdapter())
                 .registerTypeAdapter(ByteString::class.java, ByteStringBase64Adapter())
-                .registerTypeAdapter(RSAKey.Public::class.java, RSAKey.Public.GsonAdapter())
-                .registerTypeAdapter(RSAKey.Private::class.java, RSAKey.Private.GsonAdapter())
```

Comment:
GsonBuilder should be removed completely ?

Reply:
Removed all Gson references